File: src/test/java/no/nav/veilarbaktivitet/brukernotifikasjon/OpprettVarselDto.java
package no.nav.veilarbaktivitet.brukernotifikasjon;

import lombok.Data;
import no.nav.tms.varsel.action.EksternVarslingBestilling;
import no.nav.tms.varsel.action.Varseltype;

@Data
public class OpprettVarselDto {
    public String varselId;
    public String ident;
    public String tekst;
    public String link;
    public Varseltype type;
    public EksternVarslingBestilling eksternVarsling;
    public long tidspunkt;
    public Produsent produsent;
}

File: src/test/kotlin/no/nav/veilarbaktivitet/brukernotifikasjon/OpprettVarselDto.kt
package no.nav.veilarbaktivitet.brukernotifikasjon

import no.nav.tms.varsel.action.EksternVarslingBestilling
import no.nav.tms.varsel.action.Varseltype

//@Data
data class OpprettVarselDto (
    var varselId: String,
    var ident: String,
    var tekster: List<Tekst>,
    var link: String,
    var type: Varseltype,
    var eksternVarsling: EksternVarslingBestilling,
    var produsent: Produsent
)

class Tekst(
    val tekst: String,
    val spraakkode: String,
    val default: Boolean
)
